utility-emissions-channel/Gruntfile.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+module.exports = function(grunt) {
+
+    grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    
+         jshint: {
+                 options: {
+                    configuration: grunt.file.readJSON(".jshintrc"),
+                    force: false,
+                    fix: false
+                },
+            all: [
+                "Gruntfile.js",
+                "typescript/**/*.js",
+                "docker-compose-setup/egrid-data-loader.js",
+                "chaincode/node/**/*.js",
+                "emissions_calc_test/test.js"
+            ], 
+        },
+        
+        
+        tslint: {
+                options: {
+                    configuration: grunt.file.readJSON("tslint.json"),
+                    force: false,
+                    fix: false
+                },
+                files: {
+                     src: ['typescript_app/**/*.ts', 'chaincode/typescript/**/*.ts', "!typescript_app/node_modules/**/*.ts"]
+                    
+                },
+        },
+        
+       mochaTest: {
```

Comment:
Use of grunt-mocha-test for running out automated test written in mocha and chai.